message-analyzer/PostWithSentiment.cs

```diff
@@ -0,0 +1,23 @@
+﻿// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+namespace MessageAnalyzer
+{
+    internal class PostWithSentiment : Post
+    {
+        public PostWithSentiment() { }
+
+        public PostWithSentiment(Post p)
+        {
+            this.CorrelationId = p.CorrelationId;
+            this.MessageId = p.MessageId;
+            this.Message = p.Message;
+            this.CreationDate = p.CreationDate;
+        }
+
+        // like "stars": 1-5
+        public int Sentiment { get; set; }
```

Comment:
I was thinking of an enumeration: POSITIVE, NEUTRAL, NEGATIVE or VERY POSITIVE, POSITIVE, NEUTRAL, NEGATIVE, VERY NEGATIVE

Reply:
Made string with 5 values.